Python/jit.c

```diff
@@ -507,12 +507,12 @@ _PyJIT_Compile(_PyExecutorObject *executor, const _PyUOpInstruction trace[], siz
     unsigned char *code = memory;
     unsigned char *data = memory + code_size;
     state.trampolines.mem = memory + code_size + data_size;
-    // Compile the trampoline, which handles converting between the native
+    // Compile the shim, which handles converting between the native
     // calling convention and the calling convention used by jitted code
     // (which may be different for efficiency reasons). On platforms where
-    // we don't change calling conventions, the trampoline is empty and
+    // we don't change calling conventions, the shim is empty and
     // nothing is emitted here:
```

Comment:
I just realized that the comment "On platforms where we don't change calling conventions, the shim is empty and nothing is emitted here:" is incorrect. Can you remove it?